docs/historic.md

```diff
@@ -30,4 +30,24 @@ res.to_pandas().head()
 | 2022-06-01 07:00:00+00:00 |         13 |    62 |    12 |
 | 2022-06-01 07:30:00+00:00 |         13 |     0 |     0 |
 | 2022-06-01 08:00:00+00:00 |         12 |     0 |     0 |
-| 2022-06-01 08:30:00+00:00 |         12 |     0 |     0 |
\ No newline at end of file
+| 2022-06-01 08:30:00+00:00 |         12 |     0 |     0 |
+
+
+### Example of multi period request
```

Comment:
In this doco you could also cross reference the example of concurrent requests at https://solcast.github.io/solcast-api-python-sdk/notebooks/1.3%20Getting%20Data%20-%20Make%20Concurrent%20Requests/

Reply:
Added